vslib/src/SwitchStateBase.cpp

```diff
@@ -1811,6 +1815,17 @@ sai_status_t SwitchStateBase::refresh_read_only(
         }
     }
 
+    if (meta->objecttype == SAI_OBJECT_TYPE_SYSTEM_PORT)
+    {
+        switch (meta->attrid)
+        {
+            case SAI_SYSTEM_PORT_ATTR_TYPE:
+            case SAI_SYSTEM_PORT_ATTR_CONFIG_INFO:
```

Comment:
please remove SAI_SYSTEM_PORT_ATTR_CONFIG_INFO it's not READ_ONLY attribute

Reply:
Yes. This is not a read-only attribute. I added a function for refresh system port config info and return success. And also gave comment for doing so.

Reply:
if attribute is NOT read_only, it should not be in this function

Reply:
Yes. I'll remove SAI_SYSTEM_PORT_ATTR_CONFIG_INFO attribute and also remove the refresh function. Thanks

Reply:
Fixed